From ddd3d20d40b8df6c97312f8ad723ca543a08e046 Mon Sep 17 00:00:00 2001
From: Ben Cressey <bcressey@amazon.com>
Date: Sat, 10 Aug 2019 04:53:31 +0000
Subject: [PATCH 2/4] exclude unused components

The extensions are all shell scripts, which we don't support. The docs
require a native helper program to generate, which doesn't work. While
we're at it, turn off the man pages and the testing code that we won't
end up using.

Signed-off-by: Ben Cressey <bcressey@amazon.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 71c2fbb5..0278e40b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -6,7 +6,7 @@ MAINTAINERCLEANFILES		= Makefile.in aclocal.m4 config.guess config.*   \
 				  $(DIST_ARCHIVES) $(PACKAGE)-*.tar.bz2
 
 SUBDIRS				= include src util schema client server nanny \
-				  autoip4 dhcp4 dhcp6 etc extensions man doc testing
+				  dhcp4 dhcp6 etc
 
 
 pkgconfig_DATA			= wicked.pc
-- 
2.32.0

